modules/users/manifests/anabotto.pp

```diff
@@ -0,0 +1,8 @@
+# Creates the anabotto user
+class users::anabotto {
+  govuk_user { 'anabotto':
+    fullname => 'Ana Castillo Botto',
```

Comment:
```suggestion
    fullname => 'Ana Botto',
```

Unfortunately our CI requires your full name to match your email address (though I'm not sure why...)